xknx/devices/fan.py

```diff
@@ -47,13 +49,15 @@ def __init__(
         group_address_speed_state: Optional["GroupAddressableType"] = None,
         device_updated_cb: Optional[DeviceCallbackType] = None,
         mode: FanSpeedMode = DEFAULT_MODE,
+        max_step: int = 255,
```

Comment:
should we default this to `None` and imply FanSpeedMode.Step if it is set? 
This way the Device could be configured by eg. xknx.yaml from strings/ints only (the Enum wouldn't have to be resolved).

Reply:
Done